Formula/m/mpich.rb

```diff
@@ -55,15 +55,16 @@ def install
 
     args = %W[
       --disable-dependency-tracking
+      --disable-silent-rules
       --enable-fast=all,O3
       --enable-g=dbg
       --enable-romio
       --enable-shared
+      --with-hwloc=#{Formula["hwloc"].opt_prefix}
       --with-pm=hydra
       F77=gfortran
       FC=gfortran
       FCFLAGS=-fallow-argument-mismatch
```

Comment:
Side note: is this, and the `FFLAGS=-fallow-argument-mismatch` below that says "Flag for compatibility with GCC 10" still necessary? Sounds temporary with how the comment is written

Reply:
Please try removing it. I don't think it is no longer required (at least to be passed explicitly). Worst case, configure/make would fail and we would notice.